### PR TITLE
v2 release notes file

### PR DESCRIPTION
**Pull request by cfarmer-fearless**

### CODE REVIEW

File: docs/Release Notes/DEX Upload Release Notes - Version 2.1.1.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- Support backwards-compatible upload config filenames.  This allows senders migrating to v2 sender manifests to use different filenames for their v2 upload config JSON file.
+- Improved reporting to the Processing Status API.  This includes several new report types, but also a full migration to sending report messages to an Azure Service Bus instead of over HTTP.  This improves observability into files being uploaded and processed.
+- The Upload API is no longer sending trace information to the Processing Status API.
+- Improved logging.  This improves debugging and troubleshooting.
+- Support for CELR, NRSS, and EHDI programs.
+
+## New Features
+- Info endpoint.  This is a new HTTP endpoint where authenticated users can send a GET request to /upload/info/{uploadID} and get a response containing metadata about the file that was uploaded.  The uploadID path parameter is the unique ID given back by our service when an upload is complete.
+- Health check endpoint.  This is a new HTTP endpoint where users authenticated users can send a GET request to /upload/health and check the overall health of the Upload API service, and the other critical services that it depends on.
```
Comment (AJKimpo) [inline docs/Release Notes/DEX Upload Release Notes - Version 2.1.1.md:23]: you have the word 'users' twice

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
...
+- Support backwards-compatible upload config filenames.  This allows senders migrating to v2 sender manifests to use different filenames for their v2 upload config JSON file.
+- Improved reporting to the Processing Status API.  This includes several new report types, but also a full migration to sending report messages to an Azure Service Bus instead of over HTTP.  This improves observability into files being uploaded and processed.
+- The Upload API is no longer sending trace information to the Processing Status API.
+- Improved logging.  This improves debugging and troubleshooting.
+- Support for CELR, NRSS, and EHDI programs.
+- Info endpoint.  This is a new HTTP endpoint where authenticated users can send a GET request to /upload/info/{uploadID} and get a response containing metadata about the file that was uploaded.  The uploadID path parameter is the unique ID given back by our service when an upload is complete.
+- Health check endpoint.  This is a new HTTP endpoint where users authenticated users can send a GET request to /upload/health and check the overall health of the Upload API service, and the other critical services that it depends on.
```
Comment (AJKimpo) [inline CHANGELOG.md:15]: the word 'user' appears twice in this sentence

**Review by AJKimpo — CHANGES_REQUESTED** [review]
small text changes to remove redundant words

File: docs/Release Notes/DEX Upload Release Notes - Version 2.1.1.md (hunk 1 above)
Comment (cfarmer-fearless) [reply docs/Release Notes/DEX Upload Release Notes - Version 2.1.1.md]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (cfarmer-fearless) [reply CHANGELOG.md]: Fixed

**Review by AJKimpo — APPROVED** [review]
lgtm

**Pull request merged**